docker/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM node:${NODE_VERSION}
 ARG GITHUB_ACCOUNT=telefonicaid
 ARG GITHUB_REPOSITORY=iotagent-json
 ARG DOWNLOAD=latest
+ARG SOURCE_BRANCH=master
```

Comment:
Should the usage of this new ARG be taken into accoun in section "How to build your own image" of the README.md file? What do you think @jason-fox ?

Reply:
Yes - it should now be split into three parts.

see - [Ultralight](https://github.com/jason-fox/iotagent-ul/commit/ac295c05c9bffff0e3611eb85e4561b1c25243fb#diff-4b5e6a0d7df359d2d18257ca84179db7fd61ef67)


## How to build an image

--- etc

## Building from your own fork

To download code from your own fork of the GitHub repository add the  arguments to the `docker build` command.	

```console
docker build -t iot-agent . \
    --build-arg GITHUB_ACCOUNT=<your account> \
    --build-arg GITHUB_REPOSITORY=<your repo> \
    --build-arg SOURCE_BRANCH=<your branch>
```

 ## Building from your own source files

To download code from your own fork of the GitHub repository add the `GITHUB_ACCOUNT`, `GITHUB_REPOSITORY` and `SOURCE_BRANCH` arguments (default `master`) to the `docker build` command.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-json/pull/390/commits/aef402558bf56ed213c85cee45b6be416f8efdea (by imitation of https://github.com/jason-fox/iotagent-ul/commit/ac295c05c9bffff0e3611eb85e4561b1c25243fb#diff-4b5e6a0d7df359d2d18257ca84179db7fd61ef67)

Reply:
👍 LGTM